modules/core/test/v2/unit/coins/eos.ts

```diff
@@ -103,6 +106,67 @@ describe('EOS:', function() {
     });
   });
 
+  it('should create unsigned recovery transaction without Bitgo', async function() {
+    const userKey = 'xpub661MyMwAqRbcH1oUADxatLuKkVjaDB2zTNJoZQsGVQEvoogpbXJw24QMokNwFKj9Qhci6KWaCcQKrzpL4LCQXXX3YpTQxgD9KLBjhDrUWo4';
+    const backupKey = 'xpub661MyMwAqRbcH1n6sgY29G7dAxL7twS8rt1jyuuQb1kfnA7s3FJPGoVqb9JenXkeJmC4jZ8iVscn3AH6MkYAVc61FTYCHpxv5cxWar5Jw3C';
+    const rootAddress = 'i1skda3kso43';
+    const destinationAddress = 'ks13kdh245ls';
+
+    // load mock responses from json files
+    const rootResponse = JSON.parse(
```

Comment:
You don't need to use `fs.readFileSync` to load json files, you can just `import` them if they are set up as the default export - they are valid js.